plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -263,6 +268,18 @@ public VmwareManagerImpl() {
         _storageMgr = new VmwareStorageManagerImpl(this);
     }
 
+    private boolean isSystemVmIsoCopyNeeded(File srcIso, File destIso) {
+        boolean copyNeeded = !destIso.exists();
+        if (!copyNeeded) {
+            try {
+                copyNeeded = !StringUtils.equals(DigestUtils.md5Hex(new FileInputStream(srcIso)), DigestUtils.md5Hex(new FileInputStream(destIso)));
```

Comment:
Maybe also add a log line in case copyNeeded == true?

Reply:
Added